autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/utils.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any, Dict, List
+
+from sklearn.base import BaseEstimator
+
+
+def get_tabular_preprocessers(X: Dict[str, Any]) -> Dict[str, List[BaseEstimator]]:
+    preprocessor = dict(numerical=list(), categorical=list())  # type: Dict[str, List[BaseEstimator]]
+
+    for key, value in X.items():
+        if isinstance(value, dict):
+            if 'numerical' or 'categorical' in value.keys():
+                if isinstance(value['numerical'], BaseEstimator):
```

Comment:
Instead of asking for <BaseEstimator>, doesn't it makes sense to ask for <autoPyTorchPreprocessingComponent> component, just to be safe?

Reply:
now the tabular preprocessors are adding the sklearn preprocessors to the fit dictionary. Therefore the only base class they share is base estimators

Reply:
Oh, I see.. then It makes sense. Just please add a comment in code explaining this.